AI/daily_data_collection/main.py

```
@@ -0,0 +1,1165 @@
+            return None
+
+        revenue_row = find_row(fs, ["Total Revenue", "Revenue"])
+        net_income_row = find_row(fs, ["Net Income", "NetIncome"])
+        assets_row = find_row(bs, ["Total Assets"])
+        liab_row = find_row(bs, ["Total Liab", "Total Liabilities"])
+        equity_row = find_row(bs, ["Total Stockholder Equity", "Total Equity"])
+
+        info = {}
+        try:
+            info = yf_t.info or {}
+        except:
```

Ruff (0.14.6)

Do not use bare `except`

(E722)